core/chains/evm/gas/rollups/zkSync_l1_oracle.go

```diff
@@ -120,6 +120,7 @@ func (o *zkSyncL1Oracle) run() {
 func (o *zkSyncL1Oracle) refresh() (t *time.Timer) {
 	t, err := o.refreshWithError()
 	if err != nil {
+		o.logger.Criticalw("Failed to refresh gas price", "err", err)
```

Comment:
@simsonraj I had trouble debugging this PR, because these refresh methods are pushing errors to mark the service unhealthy, but without logging anything. These are typically paired with a critical log. Is that appropriate to do in this case, and the other files in this PR?

Reply:
Good catch, I think its appropriate to append an error, but not sure if it needs to be critical if it is polling. At least for zkSync case, it may not be since we dont need gasPrice exactly real-time & not sure how much impact an RPC malfunction will have here. Looking at Arbitrum it seems the intent was to log it as an error.
May be @amit-momin @FelixFan1992, any opinions?

Reply:
What if we looked at the timestamp, and only took this action (both log and unhealthy) if we cannot refresh after some threshold of time has passed?

Reply:
Logging the error definitely makes sense here. Following the [example](https://github.com/smartcontractkit/chainlink/blob/66f154745d3d973aa8b629a99a65a660a1d1b3bc/core/chains/evm/gas/block_history_estimator.go#L308) of other gas estimator code, I think it's ok to log `Critical` here. If there's an RPC issue, it's likely a connectivity issue which we'd want to surface to NOPs urgently.  Although I'm sure other components would also start making some noise for it.